project.clj

```diff
@@ -1,70 +1,43 @@
-                 ;; ring-defaults brings in a bad, old version of the servlet-api, which
-                 ;; now has a new artifact name (javax.servlet/javax.servlet-api).  If we
-                 ;; don't exclude the old one here, they'll both be brought in, and consumers
-                 ;; will be subject to the whims of which one shows up on the classpath first.
-                 ;; thus, we need to use exclusions here, even though we'd normally resolve
-                 ;; this type of thing by just specifying a fixed dependency version.
-                 [ring/ring-defaults "0.2.0" :exclusions [javax.servlet/servlet-api]]
+  :exclusions [org.clojure/clojure]
 
-                 [slingshot "0.12.2"]
-                 [trptcolin/versioneer "0.2.0"]
-                 [org.clojure/java.jmx "0.3.1"]
+  :dependencies [[org.clojure/clojure]
 
-                 [puppetlabs/kitchensink ~ks-version]
-                 [puppetlabs/trapperkeeper ~tk-version]
-                 [puppetlabs/trapperkeeper-scheduler "0.0.1"]
-                 [puppetlabs/ring-middleware "1.0.0"]
-                 [puppetlabs/comidi "0.3.1"]
-                 [puppetlabs/i18n "0.4.3"]]
+                 [cheshire]
+                 [slingshot]
+                 [prismatic/schema]
+                 [trptcolin/versioneer]
+                 [ring/ring-defaults]
+                 [org.clojure/java.jmx]
 
-  :lein-release {:scm         :git
-                 :deploy-via  :lein-deploy}
+                 [puppetlabs/kitchensink]
+                 [puppetlabs/trapperkeeper]
+                 [puppetlabs/trapperkeeper-scheduler]
+                 [puppetlabs/ring-middleware]
+                 [puppetlabs/comidi]
+                 [puppetlabs/i18n]]
 
   :deploy-repositories [["releases" {:url "https://clojars.org/repo"
                                      :username :env/clojars_jenkins_username
                                      :password :env/clojars_jenkins_password
                                      :sign-releases false}]]
 
-  :profiles {:dev {:dependencies [
-                                  ;; Begin transitive dependency resolution
-                                  [clj-time "0.11.0"]
-                                  [commons-io "2.5"]
-                                  ;; End transitive dependency resolution
-
-                                  [puppetlabs/http-client "0.5.0"]
-                                  [puppetlabs/trapperkeeper ~tk-version :classifier "test"]
-                                  [puppetlabs/trapperkeeper-webserver-jetty9 "1.5.6"]
-                                  [puppetlabs/kitchensink ~ks-version :classifier "test"]]}}
+  :profiles {:dev {:dependencies [[puppetlabs/http-client]
+                                  [puppetlabs/trapperkeeper nil :classifier "test"]
+                                  [puppetlabs/trapperkeeper-webserver-jetty9]
+                                  [puppetlabs/kitchensink nil :classifier "test"]]}}
```

Comment:
They were intentional, they were required for some versions of lein, not sure if the fix for that is in lein 2.7.1 or not.  I'll try it w/o them real quick.